extensions/wikia/AdEngine/js/AdSlotBottomLeaderboard.run.js

```diff
@@ -11,9 +11,9 @@ require([
 		return false;
 	}
 
-	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slot, provider) {
+	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slotName, provider) {
 
-		if (slot[0].indexOf('LEADERBOARD') !== -1 && (slot[2] === 'Liftium' || provider.name === 'Later')) {
+		if (slotName.indexOf('LEADERBOARD') !== -1 && (provider.name === 'Liftium' || provider.name === 'Later')) {
```

Comment:
Since it's called twice, provider.name could be cached in the local var (name?) to prevent double looping through provider object.